Makefile

```diff
@@ -26,7 +26,8 @@ test:
 integrationtests: FILE = *
 integrationtests: TEST = ''
 integrationtests:
-	cargo test --manifest-path eel/Cargo.toml --features nigiri --test '$(FILE)' -- $(TEST)
+	LSPD_HOME=lspd RGS_HOME=rgs cargo test --features nigiri --test '$(FILE)' -- $(TEST)
+	#LSPD_HOME=../lspd RGS_HOME=../rgs cargo test --manifest-path eel/Cargo.toml --features nigiri --test '$(FILE)' -- $(TEST)
```

Comment:
Do we need this comment?
```suggestion
```

Reply:
Sorry, I should not have commited this.

That's not a comment but the actual instruction to run the eel integration tests. I just commented it for local testing and then accidentally commited it.

Fixed and force-pushed rn.